cmd/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/containerd/containerd v1.7.12
 	github.com/containerd/log v0.1.0
 	github.com/coreos/go-systemd/v22 v22.5.0
-	github.com/docker/cli v25.0.0+incompatible
+	github.com/docker/cli v25.0.1+incompatible
```

Comment:
Diff https://github.com/docker/cli/compare/v25.0.0...v25.0.1

Mostly documentation updates and an unrelated bug fix.